contracts/orderbook/Enumerates/EnumTraders.sol

```diff
@@ -0,0 +1,79 @@
+pragma solidity ^0.8.4;
+
+library ActiveTraders {
```

Comment:
create a library for library EnumerableAddressSet

Reply:
actually library EnumerableBytes32Set, can accomodate all other types by length. look at `addAddress` not sure if its better concrete implementation one generic. gas wise